.rubocop.yml

```diff
@@ -28,6 +39,11 @@ Style/Next:
 Style/NonNilCheck:
   Enabled: false
 
+# This cop isn't compatible with Ruby < 2.0
+Style/SymbolArray:
+  Exclude:
+    - "Rakefile"
```

Comment:
This cop enforces to use a syntax that is not available on Ruby < 2.0.
You can see the actual error message in one of the failed tests [1].

[1] https://travis-ci.org/ghoneycutt/puppet-module-nscd/jobs/220867193#L213-L216